### ISSUE TITLE
500 error at startup

### ISSUE THREAD

**Issue by hannahlindsley** [webanno/webanno#729]
Downloaded latest release, double-clicked, and opened localhost:8080 in Chrome.

```
Status Code: 500

Exception: The error below occurred during context initialisation, so no further requests can be processed:
org.springframework.context.ApplicationContextException: Failed to start bean 'fixCoreferenceMigration'; nested exception is java.lang.NullPointerException
	at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:176)
	at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:51)
	at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:346)
	at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:149)
	at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:112)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:879)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:545)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:444)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:326)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at winstone.WebAppConfiguration.(WebAppConfiguration.java:889)
	at winstone.HostConfiguration.initWebApp(HostConfiguration.java:131)
	at winstone.HostConfiguration.(HostConfiguration.java:73)
	at winstone.HostGroup.initHost(HostGroup.java:85)
	at winstone.HostGroup.(HostGroup.java:45)
	at winstone.Launcher.(Launcher.java:196)
	at winstone.Launcher.main(Launcher.java:391)
Caused by: java.lang.NullPointerException
	at de.tudarmstadt.ukp.clarin.webanno.api.dao.migration.FixCoreferenceFeatures.doMigration(FixCoreferenceFeatures.java:103)
	at de.tudarmstadt.ukp.clarin.webanno.api.dao.migration.FixCoreferenceFeatures.start(FixCoreferenceFeatures.java:67)
	at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:173)
	... 16 more

Stacktrace:
org.springframework.context.ApplicationContextException: Failed to start bean 'fixCoreferenceMigration'; nested exception is java.lang.NullPointerException
	at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:176)
	at org.springframework.context.support.DefaultLifecycleProcessor.access$200(DefaultLifecycleProcessor.java:51)
	at org.springframework.context.support.DefaultLifecycleProcessor$LifecycleGroup.start(DefaultLifecycleProcessor.java:346)
	at org.springframework.context.support.DefaultLifecycleProcessor.startBeans(DefaultLifecycleProcessor.java:149)
	at org.springframework.context.support.DefaultLifecycleProcessor.onRefresh(DefaultLifecycleProcessor.java:112)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:879)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:545)
	at org.springframework.web.context.ContextLoader.configureAndRefreshWebApplicationContext(ContextLoader.java:444)
	at org.springframework.web.context.ContextLoader.initWebApplicationContext(ContextLoader.java:326)
	at org.springframework.web.context.ContextLoaderListener.contextInitialized(ContextLoaderListener.java:107)
	at winstone.WebAppConfiguration.(WebAppConfiguration.java:889)
	at winstone.HostConfiguration.initWebApp(HostConfiguration.java:131)
	at winstone.HostConfiguration.(HostConfiguration.java:73)
	at winstone.HostGroup.initHost(HostGroup.java:85)
	at winstone.HostGroup.(HostGroup.java:45)
	at winstone.Launcher.(Launcher.java:196)
	at winstone.Launcher.main(Launcher.java:391)
Caused by: java.lang.NullPointerException
	at de.tudarmstadt.ukp.clarin.webanno.api.dao.migration.FixCoreferenceFeatures.doMigration(FixCoreferenceFeatures.java:103)
	at de.tudarmstadt.ukp.clarin.webanno.api.dao.migration.FixCoreferenceFeatures.start(FixCoreferenceFeatures.java:67)
	at org.springframework.context.support.DefaultLifecycleProcessor.doStart(DefaultLifecycleProcessor.java:173)
	... 16 more

Generated by Winstone Servlet Engine v0.9.10 at Tue Nov 21 10:02:07 CST 2017
```

**Comment by reckart** [comment]
Which version of WebAnno are you using?

**Comment by hannahlindsley** [comment]
WebAnno 3.2.4 standalone

**Comment by reckart** [comment]
I'm afraid that WebAnno presently doesn't work well (or at all) on a Java 9 JRE/JDK. I could reproduce the issue you describe on Java 9. But when I switch to Java 8 before starting WebAnno, it starts up fine.

**Comment by hannahlindsley** [comment]
Oh, great! Thanks, I'll switch over to that.

**Comment by reckart** [comment]
We're looking for feedback on the latest beta. Feel invited to check out 3.3.0-beta-6 from here: https://webanno.github.io/webanno/downloads-beta/

**Comment by hannahlindsley** [comment]
Ok, will do.

**Comment by reckart** [comment]
Fixed via #656.
